src/commands/blocks.ts

```
@@ -274,18 +275,18 @@ export const blocksCommand = define({
-									return limit != null
-										? {
-												limit,
-												projectedUsage: projection.totalTokens,
-												percentUsed: (projection.totalTokens / limit) * 100,
-												status: projection.totalTokens > limit
-													? 'exceeds'
-													: projection.totalTokens > limit * BLOCKS_WARNING_THRESHOLD ? 'warning' : 'ok',
-											}
-										: undefined;
-								})()
+								const limit = parseTokenLimit(ctx.values.tokenLimit, maxTokensFromAll);
```

ESLint

[error] Expected indentation of 9 tabs but found 8.

(style/indent)